tapeout/src/main/scala/transforms/Generate.scala

```diff
@@ -218,8 +218,8 @@ sealed trait GenerateTopAndHarnessApp extends LazyLogging { this: App =>
       case x: FirrtlExecutionSuccess =>
         dump(x, tapeoutOptions.topFir, tapeoutOptions.topAnnoOut)
         x.circuitState.circuit.modules.collect{ case e: ExtModule => e }
-      case _ =>
-        throw new Exception("executeTop failed on illegal FIRRTL input!")
+      case e =>
```

Comment:
just a nit, but we should probably use the same case variable (`e` vs `x`) for both cases for clarity- one `e` is an `ExtModule` and the other is a result